addons/sys_external/fnc_getExternalUseStatus.sqf

```diff
@@ -0,0 +1,37 @@
+/*
+ * Author: ACRE2Team
+ * Retrieves the external use status of a radio
+ *
+ * Arguments:
+ * 0: Unique radio identity <STRING>
+ *
+ * Return Value:
+ * Array with information if a radio is shared, is in external use, the owner and the end user <ARRAY>
+ *
+ * Example:
+ * ["ACRE_PRC343_ID_1"] call acre_sys_external_getExternalUseStatus
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_radioId"];
+
+private _isShared = [_radioId] call FUNC(isRadioShared);
+
+private ["_isUsedExternally", "_owner", "_user"];
```

Comment:
This should be mixed into the below lines
i.e. `private _isUsedExternally = false;`